build.xml

```diff
@@ -1,45 +0,0 @@
-<project name="user-agent-tools" default="compile" basedir=".">
```

Comment:
Why did you remolve this file? I'd rather use ant for my personal workflow, can't both coexist ?